Tasks/AzurePowerShellV2/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 2,
         "Minor": 1,
-        "Patch": 4
+        "Patch": 5
```

Comment:
Can you check if these versions are ok to be published and consumed for 143 branch?

Reply:
M143 version `2.1.4`
M144 version `2.1.6`
We are good to add this version.